packages/daimo-common/package.json

```diff
@@ -27,6 +27,7 @@
     "@noble/curves": "^1.2.0",
     "@scure/base": "^1.1.3",
     "@scure/bip39": "^1.3.0",
+    "@uniswap/sdk-core": "^4.2.1",
```

Comment:
this is a heavy dependency, & looks like we're using it just for the WETH addresses.

recommend hardcoding those (which we're already doing for DAI, USDbC etc)

minimizing dependencies important everywhere but esp for `daimo-common` since it gets imported everywhere else, including web and mobile